bundles/org.eclipse.swt/Eclipse SWT PI/win32/library/build.bat

```diff
@@ -152,14 +134,6 @@ GOTO :EOF
 	@rem If the folder isn't there, then skip it without printing errors
 	IF NOT EXIST "%TESTED_VS_PATH%" GOTO :EOF
 
-	@rem Try this path
-	CALL :TryToUseVisualStudio "%TESTED_VS_PATH%"
-GOTO :EOF
-
-@rem Test Visual Studio and set '%MSVC_HOME%' on success
-@rem %1 = tested path
-:TryToUseVisualStudio
```

Comment:
The point of separate function here was for code clarity, to distinguish from 1/2/3 searching layers

Reply:
The three layers are still distinguished altough there are actually only two layers, one for the Version and one for the edition. The last one only assembles the path and checks if its's valid. And having that very last part in a separate method did not seem to improve the readability for me but still added more code.
If you want to have a method named `TryToUseVisualStudio` I would suggest to rename `FindVisualStudio3` to it, so that FindVisualStudio1 iterates over the versions, FindVisualStudio2 over the editon and the last tries to use that combination.

Reply:
It's a small issue and I'll leave it to you. Still, I find it not worthy to add diffs to patch over a matter of tastes.

Reply:
> It's a small issue and I'll leave it to you. Still, I find it not worthy to add diffs to patch over a matter of tastes.

OK, restored that part.